dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -186,6 +198,73 @@ void CommandBarFlyoutCommandBar::PlayCloseAnimation(std::function<void()> onComp
     }
 }
 
+void CommandBarFlyoutCommandBar::UpdateFlowsFromAndFlowsTo()
+{
+    if (m_currentPrimaryItemsEndElement)
+    {
+        winrt::AutomationProperties::GetFlowsTo(m_currentPrimaryItemsEndElement.get()).Clear();
+        m_currentPrimaryItemsEndElement.set(nullptr);
+    }
+
+    if (m_currentSecondaryItemsStartElement)
+    {
+        winrt::AutomationProperties::GetFlowsFrom(m_currentSecondaryItemsStartElement.get()).Clear();
+        m_currentSecondaryItemsStartElement.set(nullptr);
+    }
+
+    // If we're not open, then we don't want to do anything special - the only time we do need to do something special
+    // is when the secondary commands are showing, in which case we want to connect the primary and secondary command lists.
+    if (IsOpen())
+    {
+        auto isElementFocusable = [](winrt::ICommandBarElement const& element)
+        {
+            winrt::Control primaryCommandAsControl = element.try_as<winrt::Control>();
+            return
+                primaryCommandAsControl &&
+                primaryCommandAsControl.Visibility() == winrt::Visibility::Visible &&
+                (primaryCommandAsControl.IsEnabled() || primaryCommandAsControl.AllowFocusWhenDisabled()) &&
+                primaryCommandAsControl.IsTabStop();
+        };
+
+        // If we have a more button, then that's the last element in our primary items list.
+        // Otherwise, we'll find the last focusable element in the primary commands.
+        if (m_moreButton)
+        {
+            m_currentPrimaryItemsEndElement.set(m_moreButton.get());
+        }
+        else
+        {
+            auto primaryCommands = PrimaryCommands();
+            for (int i = static_cast<int>(primaryCommands.Size() - 1); i >= 0; i--)
+            {
+                auto primaryCommand = primaryCommands.GetAt(i);
+                if (isElementFocusable(primaryCommand))
```

Comment:
It looks like STL algorithms don't work since it's an IObservableVector rather than an STL vector.  I can make the SecondaryCommands loop a range-based for loop, though.